ciphers/rsa_key_generator.py

```diff
@@ -12,7 +12,7 @@ def main():
     print("Key files generation successful.")
 
 
-def generateKey(keySize):
+def generateKey(keySize: int) -> (tuple, tuple):
```

Comment:
Or `tuple[int, int]`?

Reply:
yes, probably should write it more explicitly .